### PR TITLE
maintainers: add kiara as maintainer

### PR DESCRIPTION
**Pull request by KiaraGrouwstra**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by FliegendeWurst — APPROVED** [review]

**Comment by OPNA2608** [comment]
`ofborg-eval-lib-tests` fails:

```
       error: The option `lib.maintainers.kiara.handle' does not exist. Definition values:
       - In `/ofborg/checkout/1/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-4/maintainers/maintainer-list.nix': "kiara"
```

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -11429,6 +11429,14 @@
     name = "Khushraj Rathod";
     keys = [ { fingerprint = "1988 3FD8 EA2E B4EC 0A93  1E22 B77B 2A40 E770 2F19"; } ];
   };
+  kiara = {
+    name = "kiara";
+    handle = "kiara";
```
Comment (OPNA2608) [inline maintainers/maintainer-list.nix:11434]: This is not a valid attribute here: https://github.com/NixOS/nixpkgs/blob/50cff47c417ca9530671786e14038d39a9e08762/maintainers/maintainer-list.nix#L5-L16

The handle you want to use is the attribute name under which you put this attrset, so `kiara = { /* ... */ };` means `kiara` is already your handle.

```suggestion
```

**Review by OPNA2608 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (KiaraGrouwstra) [reply maintainers/maintainer-list.nix]: thanks! i pushed an update now to address this.

**Comment by r-vdp** [comment]
Usually someone only gets added to this file when they actually maintain at least one package. If you are planning to adopt a package, you can just add this commit to that PR.

**Comment by KiaraGrouwstra** [comment]
@r-vdp i was splitting this off as per https://github.com/NixOS/nixpkgs/pull/354765#pullrequestreview-2425465669

**Comment by r-vdp** [comment]
> @r-vdp i was splitting this off as per https://github.com/NixOS/nixpkgs/pull/354765#pullrequestreview-2425465669

Ah, I think they meant to split it into two commits in the same PR, not into two PRs.

**Pull request closed without merging**